openig-core/src/test/java/org/forgerock/openig/filter/SwitchFilterTest.java

```diff
@@ -61,11 +61,11 @@ public class SwitchFilterTest {
     @BeforeMethod
     public void setUp() throws Exception {
         MockitoAnnotations.initMocks(this);
-        when(terminalHandler.handle(any(Context.class), any(Request.class)))
+        when(terminalHandler.handle(any(Context.class), any()))
                 .thenReturn(Promises.<Response, NeverThrowsException>newResultPromise(new Response(Status.OK)));
-        when(handler1.handle(any(Context.class), any(Request.class)))
+        when(handler1.handle(any(Context.class), any()))
                 .thenReturn(Promises.<Response, NeverThrowsException>newResultPromise(new Response(Status.OK)));
-        when(handler2.handle(any(Context.class), any(Request.class)))
+        when(handler2.handle(any(Context.class), any()))
                 .thenReturn(Promises.<Response, NeverThrowsException>newResultPromise(new Response(Status.OK)));
         when(handler3.handle(any(Context.class), any(Request.class)))
```

Comment:
Hi @aldaris,
Thanks for the contribution!
Could you please explain why you removed the `Request.class` from the argument?
I think we should keep it to ensure the `handle` function is called with the `Request` type.

Reply:
There was a change in the Mockito API across the versions: any(Request.class) previously matched null as well, but now it doesn't. any() however matches null as well. The tests in question are passing in null as a request, so these mock setups stopped working as expected.